dashboard/app/models/lesson.rb

```diff
@@ -505,10 +505,12 @@ def related_csf_lessons
   def summarize_related_lessons
```

Comment:
Is there a test for this that needs to be updated?

Reply:
Great catch Dani, the summarize method had no test coverage on its actual output and when I went to add it I uncovered a subtle bug which led to a bunch of rejiggering.